src/ServiceStack/DependencyInjection/DependencyService.cs

```diff
@@ -75,25 +137,7 @@ public void UpdateRegistrations()
                 _containerBuilder = new ContainerBuilder();
             }
         }
-
-        private void SetSharing(IRegistrationBuilder<object, object, SingleRegistrationStyle> registration,
-                                Sharing sharing)
-        {
-            switch (sharing)
-            {
-                case Sharing.None:
-                    registration.InstancePerDependency();
-                    break;
-                case Sharing.PerRequest:
-                    registration.InstancePerMatchingLifetimeScope("httpRequest");
-                    break;
-                case Sharing.Singleton:
-                    registration.RegistrationData.Sharing = InstanceSharing.Shared;
-                    registration.RegistrationData.Lifetime = new RootScopeLifetime();
```

Comment:
:+1: Looks like its the same https://github.com/autofac/Autofac/blob/98d37597404e02aa783ca334ba8db4960cae7137/Core/Source/Autofac/Builder/RegistrationBuilderOfTLAR.cs#L110-L115